docs/plugins/backup.md

```diff
@@ -0,0 +1,56 @@
+# Moonshot backup plugin
+
+Moonshot plugin for backing up config files.
+
+## Functionality
+
+The plugin collects and deflates certain files to a single tarball,
+and uploads that to a a given S3 bucket. The whole process happens
+in memory, nothing is written to disk. The plugin currently supports single files only, 
+including whole directories in your tarball is not possible yet.
+
+The plugin uses the Moonshot AWS config, meaning that the bucket must be
```

Comment:
I might just omit this line if I were you. I don't know if we added doctor support for plugins, but if we did (or you felt like adding it) this plugin could *check* if a user has read/write access to the bucket when you run `bin/<tool> doctor`.